cmd/svcat/class/create_cmd.go

```diff
@@ -47,6 +52,12 @@ func NewCreateCmd(cxt *command.Context) *cobra.Command {
 		"Name from an existing class that will be copied (Required)",
 	)
 	cmd.MarkFlagRequired("from")
+	cmd.Flags().StringVarP(&createCmd.Namespace, "namespace", "n", "",
```

Comment:
FYI, these lines will be replaced by calling the AddNamespaced and AddScopedFlags functions mentioned above.